### PR TITLE
Remove unneeded label on bracers of strength

### PR DESCRIPTION
**Pull request by rectulo**

### CODE REVIEW

File: packs/equipment/bracers-of-strength.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
             },
             {
                 "key": "FlatModifier",
-                "label": "Belt of Giant Strength (lift heavy objects, Escape, Force Open)",
+                "label": "Bracers of Strength (lift heavy objects, Escape, Force Open)",
```
Comment (simonward) [inline packs/equipment/bracers-of-strength.json:49]: This should be either localized, or removed with a slug added.

**Comment by simonward** [comment]
Consider adding a damage link to the description:
`@Damage[(@actor.abilities.str.mod)[bludgeoning]]`

File: packs/equipment/bracers-of-strength.json (hunk 1 above)
Comment (TikaelSol) [reply packs/equipment/bracers-of-strength.json:49]: The slug isn't optional if there's any overlap in selectors, even if the type is different.

Comment (TikaelSol) [inline packs/equipment/bracers-of-strength.json:49]:
```suggestion
```

**Pull request merged**